public/js/controllers/game.js

```diff
@@ -121,7 +125,11 @@ angular.module('mean.system')
     };
 
     $scope.startGame = function() {
-      game.startGame();
+      if (game.players.length >= game.playerMinLimit) {
```

Comment:
Extract game.players.length >= game.playerMinLimit into a boolean variable and give it a meaningful name e.g isUptoRequiredNumber. This makes the if statement more readable. Afterwards, a ternary operator would also be more succinct and cleaner.